frontend/src/components/Layout/Menu/styled.ts

```diff
@@ -1,7 +1,6 @@
-import styled from 'styled-components';
-import { makeStyles } from '@material-ui/core';
-import { ListItemButton, ListItemText } from '@mui/material';
 import { ReactFragment } from 'react';
+import { styled, ListItemButton, ListItemText } from '@mui/material';
+import { makeStyles } from '@material-ui/core';
```

Comment:
please use import from mui instead

Reply:
For example: https://mui.com/material-ui/api/list-item-button/

Imports: 
![image](https://user-images.githubusercontent.com/61495158/163928566-4d6ce76b-a74c-4a53-806c-6ed4030b1016.png)